content/en/apps/reference/tasks.md

```diff
@@ -223,6 +223,55 @@ module.exports = {
 };
 ```
 
+## Default resolvedIf method
+If you skip defining `resolvedIf` section in the task definition, it will default to defaultResolvedIf method which is equivalent to this:
+
+```js
+resolvedIf: function (c, r, event, dueDate) {
+  let start = 0;
+
+  //The first report type action's form 
+  var resolvingForm = taskDefinition.actions.find(action => action.type === 'report').form;
+
+  if (r) {//Report based task
+    //Either "start of the task window" or "after the report's reported date", whichever comes later
+    start = Math.max(Utils.addDate(dueDate, -event.start).getTime(), r.reported_date + 1);
+  }
+  else {//Contact based task
+    //Start of the task window
+    start = Utils.addDate(dueDate, -event.start).getTime();
+  }
+
+  //End of the task window
+  const end = Utils.addDate(dueDate, event.end + 1).getTime();
+
+  //Returns true if the contact has at least one report with matching form type
+  //that is submitted between start and end times (inclusive)
+  return Utils.isFormSubmittedInWindow(
+    c.reports,
+    resolvingForm,
+    start,
+    end
+  );
+},
+```
+
+You can also use `this.defaultResolvedIf` directly:
+
+```js
+resolvedIf: this.defaultResolvedIf,//Same as skipping resolvedIf
+```
+
+It is also possible to specify a resolving form type:
+
+```js
+resolvedIf: function (c, r, event, dueDate) {
```

Comment:
Please replace 'c' for 'contact' and 'r' for 'report', trying to fix the one letter names for more descriptive names. Thanks!

Remember to add semi-colon at the end of each JS line. 

And use `cons` and `let` instead of `var`, basically leaving the best best practices in the code sample for others to replicate 💪